quantinuum-hugr/src/utils.rs

```diff
@@ -85,6 +85,10 @@ pub(crate) mod test_quantum_extension {
             )
             .unwrap();
 
+        extension
+            .add_op(SmolStr::new_inline("CZ"), "CZ".into(), two_qb_func())
+            .unwrap();
+
```

Comment:
worth adding this rather than just using CX?

Reply:
I wanted to match the motivating example in the test, but we can't do it without also adding a classically controlled operation.

I'll drop this.